modules/microshift-examples-of-usecase.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-examples-of-usecase_{context}"]
+= Use cases for resource manifest deletion
+
+The following explains the use case in which the resource manifest deletion is used.
+
+[id="data-removal-scenario_{context}"]
+== Data removal scenario
+Use the following procedure in the data removal scenario to completely delete the resource defined in the kustomize manifest file.
+
+.Procedure
+
+. Identify the file that needs to be placed in the `delete` subdirectories.
+. Move the manifest file into one of the `delete` subdirectories  under the configured manifests directory by running the following command:
++
+[source,terminal]
+----
+[sudo] mv /usr/lib/microshift/manifests.d/some-user-manifest /usr/lib/microshift/manifests.d/delete/some-user-manifest
+----
+. {microshift-short} detects and removes the resource after the manifest file is placed in the `delete` subdirectories.
+
+
+[id="upgrade-scenario_{context}"]
+== Upgrade scenario
+Use the following procedure in the upgrade scenario where some resources need to be removed while retaining others to preserve data.
+
+.Procedure
+
+. Identify the manifest file that requires updating. It is not necessary to include the entire resource `spec` in the `delete` subdirectories.
+. Specify the metadata, such as `name` and `namespace` of the resource to be removed, in the `delete` subdirectories.
```

Comment:
I would say:
- identify the manifest (without file, as it's most likely a directory)
- create new manifest for resource deletion that need to be deleted (`spec` can be skipped) in path like manifests.d/delete/my-manifest-delete/{kustomization.yaml,...}
- create new manifest to apply in manifests.d/my-manifest

Reply:
done